### ISSUE TITLE
Widget links open with Edge

### ISSUE THREAD

**Issue by 1111Al** [MarcGuiselin/wedge#8]
There is always the microsoft-edge:// URI scheme interception code from widgets, which always try to open with Edge. In my case it doesn't open anything, 'cause I totally disabled Edge. That setting cannot changed to my default browser.
![Senza titolo](https://github.com/MarcGuiselin/wedge/assets/122782530/8ed33f97-e171-4b35-b3b4-c0c36de5dd93)

**Comment by rcmaehl** [comment]
> In order for Wedge to work, you need to have Edge installed. Wedge will automatically intercept `microsoft-edge://` uris and forward them to your default browser without you needing to do anything. You don't need to change the settings in the screenshot you shared.
> 
> I'd recommend against uninstalling Edge since it's a system dependency and is required for apps that have web embeds. If you used a tool to uninstall Edge, it should be possible to undo using the same tool. Then you should restart your pc and re-install Wedge. Everything should then work as intended.
> 
> If you want to keep edge uninstalled you will need to create a shim. Make sure you are able to see file extensions ([guide on how to do this](https://www.elevenforum.com/t/show-or-hide-file-name-extensions-for-known-file-types-in-windows-11.898/)). Create a txt file (right click mouse in file explorer, New > Text document). Rename it to `msedge.exe` and place it in `C:\Program Files (x86)\Microsoft\Edge\Application\`. Then reinstall Wedge. I haven't tried this myself, but it should work.
> 
> Let me know if you have any success solving the issue by following either option.

I don't believe this works as I last I checked the file signature is verified which is why Aveyo and I use IE_to_Edge_stub

**Comment by 1111Al** [comment]
I haven't completely uninstalled Edge. I've worked selectively and manually on processes, services and registry keys aimed at its total deactivation. I also deleted some of its large folders that were needlessly taking up space on my SSD. Furthermore, processes related to the Edge core, such as search from the start menu, webview and other still work perfectly. Thay's why I wrote about selective deactivation. It took some time and work, but now I'm satisfied on the result. About for the problem with Wedge and that protocol, I don't know if it's tha the cause,, because I had it even before proceeding with the total deactivation of Edge.

**Comment by moinkhanif** [comment]
> In order for Wedge to work, you need to have Edge installed. Wedge will automatically intercept `microsoft-edge://` uris and forward them to your default browser without you needing to do anything. You don't need to change the settings in the screenshot you shared.
> 
> I'd recommend against uninstalling Edge since it's a system dependency and is required for apps that have web embeds. If you used a tool to uninstall Edge, it should be possible to undo using the same tool. Then you should restart your pc and re-install Wedge. Everything should then work as intended.
> 
> If you want to keep edge uninstalled you will need to create a shim. Make sure you are able to see file extensions ([guide on how to do this](https://www.elevenforum.com/t/show-or-hide-file-name-extensions-for-known-file-types-in-windows-11.898/)). Create a txt file (right click mouse in file explorer, New > Text document). Rename it to `msedge.exe` and place it in `C:\Program Files (x86)\Microsoft\Edge\Application\`. Then reinstall Wedge. I haven't tried this myself, but it should work.
> 
> Let me know if you have any success solving the issue by following either option.

This does not work unfortunately. 😢 Recently tried it and event the search menu links did not open. Atlease when original msedge in available these links open in my desired browser. Its just that the widget feed links always open edge
